packages/list-view/lib/main.js

```diff
@@ -11,5 +11,5 @@ Ember.ListItemView         = ListItemView;
 Ember.ListView             = ListView;
 Ember.ListViewHelper       = ListViewHelper;
 
-Ember.Handlebars.registerHelper('ember-list', EmberList);
-Ember.Handlebars.registerHelper('ember-virtual-list', EmberVirtualList);
+Ember.HTMLBars.registerHelper('ember-list', EmberList);
```

Comment:
This has been moved to be a private API (in https://github.com/emberjs/ember.js/pull/10379).